dub.json

```diff
@@ -4,7 +4,7 @@
 		"Paul Backus"
 	],
 	"dependencies": {
-		"sumtype": ">=0.6.2 <1.0.0"
+		"sumtype": "1.1.4"
```

Comment:
dependency should use `~>1.0` or similar to allow applications and libraries depending on this library to choose sumtype versions more freely

`~>1.0` is equal to `>=1.0.0 <2.0.0`